README.md

```diff
@@ -1,421 +1,419 @@
+  StartBlobCopyOperation operation = blockBlobClient.StartCopyFromUri(sourceUri);
+
+  // Waits for the operation to finish, checking for status every 1 second.
+  auto copyResponse = operation.PollUntilDone(std::chrono::milliseconds(1000));
+  auto propertiesModel = copyResponse.Value;
+
+  // Now you can look at API specific members on the result object that is returned.
+  if (propertiesModel.CopySource.HasValue())
+  {
+    std::cout << "The source of the copied blob is: " << propertiesModel.CopySource.Value()
+              << std::endl;
+  }
+```
+
+#### Interacting with Azure SDK for C++
+
+Static SDK members should not be accessed and SDK functions should not be called before the static initialization phase is finished.
+
+#### Visual Studio - CMakeSettings.json
+
+When building your application via Visual Studio, you can create and update a `CMakeSettings.json` file and include the following properties to let Visual Studio know where the packages are installed and which triplet needs to be used:
+
+```json
+{
+  "configurations": [
+    {
+        "cmakeToolchain": "<path to vcpkg repo>/vcpkg/scripts/buildsystems/vcpkg.cmake",
+        "variables": [
+        {
+          "name": "VCPKG_TARGET_TRIPLET",
+          "value": "x64-windows",
+          "type": "STRING"
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Azure Requirements
+
+To call Azure services, you must first have an Azure subscription. Sign up for a [free trial](https://azure.microsoft.com/pricing/free-trial/) or use your [MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/).
+
+## Packages available
+
+Each service might have a number of libraries available. These libraries follow the [Azure SDK Design Guidelines for C++](https://azure.github.io/azure-sdk/cpp_introduction.html) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure::Core](https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/README.md).
+
+The client libraries can be identified by the naming used for their folder, package, and namespace. Each will start with `azure`, followed by the service category, and then the name of the service. For example `azure-storage-blobs`.
+
+For a complete list of available packages, please see the [latest available packages](https://azure.github.io/azure-sdk/releases/latest/#c) page.
+
+> NOTE: If you need to ensure your code is ready for production we strongly recommend using one of the stable, non-beta libraries.
+
+### Vcpkg
+
+The following SDK library releases are available on [vcpkg](https://github.com/microsoft/vcpkg):
+
+- `azure-core-cpp`
+- `azure-identity-cpp`[](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md004.md)
```

Comment:
Why is this a link to someone's private repo?

Reply:
Good point. It's the comment from the markdown linter being used that accidentally showed up. Fixing.